### PR TITLE
add one-liner five node network

### PR DESCRIPTION
**Pull request by danlaine**

### CODE REVIEW

File: examples/local/indepth/main.go [hunk 1]
```diff
@@ -43,13 +43,15 @@ func shutdownOnSignal(
...
-// - add a new node
-// - remove a node
-// - stop the network
+// Shows example usage of the Avalanche Network Runner.
+// Creates a local five node Avalanche network
+// and waits for all nodes to become healthy.
+// Then, we:
+// * print the names of the nodes
+// * print the node ID of one node
+// * start a new node
```
Comment (holisticode) [inline examples/local/indepth/main.go:52]: Do these comments apply for this runner?

File: examples/local/indepth/main.go [hunk 2]
```diff
@@ -43,13 +43,15 @@ func shutdownOnSignal(
...
+// * remove an existing node
```
Comment (holisticode) [inline examples/local/indepth/main.go:53]: Same

File: examples/local/indepth/main.go [hunk 3]
```diff
@@ -43,13 +43,15 @@ func shutdownOnSignal(
...
-// - get a node and make an API call to it
-// - add a new node
-// - remove a node
-// - stop the network
+// Shows example usage of the Avalanche Network Runner.
+// Creates a local five node Avalanche network
+// and waits for all nodes to become healthy.
+// Then, we:
+// * print the names of the nodes
+// * print the node ID of one node
```
Comment (holisticode) [inline examples/local/indepth/main.go:51]: This comment should probably be changed

**Review by holisticode — COMMENTED** [review]
Looks good to me, but it appears some comments should be changed?

**Comment by danlaine** [comment]
@holisticode I'm not sure I know what you mean about the comments for `indepth`'s `main` function. I think the comments are right. Here's my output:

```
➜  avalanche-network-runner git:(add-one-liner-give-node-network) go run examples/local/indepth/main.go        
INFO [12-08|22:26:41] r/local/network.go#171: creating network with 5 nodes
INFO [12-08|22:26:41] r/local/network.go#380: adding node "node-0" with tmp dir at /tmp/avalanchego-network-runner-3216154663, logs at /tmp/avalanchego-network-runner-3216154663/logs, DB at /tmp/avalanchego-network-runner-3216154663, P2P port 40937, API port 38299
INFO [12-08|22:26:41] r/local/network.go#380: adding node "node-1" with tmp dir at /tmp/avalanchego-network-runner-1569123297, logs at /tmp/avalanchego-network-runner-1569123297/logs, DB at /tmp/avalanchego-network-runner-1569123297, P2P port 44637, API port 44497
INFO [12-08|22:26:41] r/local/network.go#380: adding node "node-2" with tmp dir at /tmp/avalanchego-network-runner-533883733, logs at /tmp/avalanchego-network-runner-533883733/logs, DB at /tmp/avalanchego-network-runner-533883733, P2P port 39521, API port 39237
INFO [12-08|22:26:41] r/local/network.go#380: adding node "node-3" with tmp dir at /tmp/avalanchego-network-runner-2098101222, logs at /tmp/avalanchego-network-runner-2098101222/logs, DB at /tmp/avalanchego-network-runner-2098101222, P2P port 41005, API port 37735
INFO [12-08|22:26:41] r/local/network.go#380: adding node "node-4" with tmp dir at /tmp/avalanchego-network-runner-3048180218, logs at /tmp/avalanchego-network-runner-3048180218/logs, DB at /tmp/avalanchego-network-runner-3048180218, P2P port 34139, API port 39935
INFO [12-08|22:26:41] r/examples/local/indepth/main.go#99: waiting for all nodes to report healthy...
INFO [12-08|22:27:02] r/examples/local/indepth/main.go#109: current network's nodes: [node-0 node-1 node-2 node-3 node-4]
INFO [12-08|22:27:02] r/examples/local/indepth/main.go#122: one node's ID is: NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg
INFO [12-08|22:27:03] r/local/network.go#380: adding node "New Node" with tmp dir at /tmp/avalanchego-network-runner-322543152, logs at /tmp/avalanchego-network-runner-322543152/logs, DB at /tmp/avalanchego-network-runner-322543152, P2P port 46233, API port 34305
INFO [12-08|22:27:03] r/examples/local/indepth/main.go#143: removing node "node-3"
INFO [12-08|22:27:03] r/examples/local/indepth/main.go#152: waiting for updated network to report healthy...
INFO [12-08|22:27:48] r/examples/local/indepth/main.go#163: updated network's nodes: [node-4 New Node node-0 node-1 node-2]
INFO [12-08|22:27:48] r/examples/local/indepth/main.go#164: Network will run until you CTRL + C to exit...
```

**Review by felipemadero — APPROVED** [review]
lgtm

**Pull request merged**